tests/testImportCentroidAlgorithm.py

```diff
@@ -59,11 +60,11 @@ def testApplyCentroid(self):
         """Test that we can instantiate and play with SillyMeasureCentroid"""
 
         for imageFactory in (
-                             afwImage.MaskedImageF,
-                             ):
+            afwImage.MaskedImageF,
```

Comment:
I strongly considered condensing it to one line, but assumed the designer anticipated adding more in the future. I'd be happy to change it.